pricefeed/pricefeed_test.go

```diff
@@ -16,12 +17,22 @@ func init() {
 	logging.SetLogLevel("pricefeed", "debug")
 }
 
+func getMainnetEndpoint() string {
+	endpoint := os.Getenv("ETH_MAINNET_ENDPOINT")
+	if endpoint == "" {
+		endpoint = mainnetEndpoint
+	}
+	return endpoint
+}
```

Comment:
Feel free to suggest a different environment variable name. I wanted to lower the friction to getting a reliable ethereum mainnet endpoint set by github action secrets, since the existing tests are failing from time to time.